namespace.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: GKE_NAMESPACE
\ No newline at end of file
+  name: KUBECTL_NAMESPACE
```

Comment:
Is this a placeholder ? Or do you intend the user to replace this? It's not a helm chart. Not clear.

Reply:
It is placeholder replaced in bash script https://github.com/andrewkandzuba/openexchange-webservice-go/blob/google-cloud-build/deploy.sh#L4

Reply:
ok